content/_index.md

```diff
@@ -4,13 +4,14 @@ hide_title: true
 images:
 - https://odin-lang.org/images/logo-slim.png
 ---
-<center id="home-info">
-<a href="//odin-lang.org">
-	<div style="max-width: 20em"><img alt="Odin Programming Language" src="/images/logo-slim.png"></div>
-</a>
+<center id="hero">
+	<a href="//odin-lang.org">
+		<div style="max-width: 20em"><img class="center" alt="Odin Programming Language" src="/images/logo-slim.png"></div>
+	</a>
+	<p id="hero-text">A fast, concise, readable, pragmatic and open sourced programming language.</p>
```

Comment:
There is also way too much vertical space between the image and the text.

Again, try not to add `center` in the markup.

Reply:
I did not add center. Look at your current code.

As-is, it's smashed together. Breathing room is less anxious.

Reply:
The breathing room is way too much still. Sorry I am being picky.